python/llm/src/bigdl/llm/ggml/model/chatglm/__init__.py

```diff
@@ -18,3 +18,5 @@
 # physically located elsewhere.
 # Otherwise there would be module not found error in non-pip's setting as Python would
 # only search the first bigdl package and end up finding only one sub-package.
+
+from .chatglm import ChatGLM
```

Comment:
others LGTM, this can cause error when `import bigdl.llm` on linux, because we have not released chatglm binary file on linux